en/micro-integrator/docs/references/connectors/file-connector/file-connector-example.md

```diff
@@ -0,0 +1,138 @@
+# File Connector Example
+
+The File Connector allows you to connect to different file systems and perform various operations with the file systems. The file connector uses the Apache Commons VFS I/O functionalities to execute operations.
+
+WSO2 EI File connector introduces the atomic operation related to the file system and allows you to easily manipulate files based on your requirement. The file streaming functionality using Apache Commons I/O lets you copy large files and reduces the file transfer time between two file systems resulting in a significant improvement in performance that can be utilized in file operations.
+
+Using File Connector it can be perform operations in local file system as well as in a remote server such as FTP and SFTP. 
+
+## What you'll build
+
+This example explains how to use File Connector to create a file in the local file system and read the particular file. The user will send a payload specifying which content to be written to the file. Based on that content a file will be created in the specified location. Then it can be read the content of the file as an http response by invoking the other api resource upon the existence of the file. 
+
+It will have two http api resources which are create and read. 
+
+* /create : It will create a file with the content that the user specifies in the payload. 
+<p><img src="../../../assets/img/connectors/FileConnector-03.png" title="Adding a Rest API" width="800" alt="Adding a Rest API"/></p>
+
+* /read : It will first check if the file exists. If so it will read the content of the file. 
+<p><img src="../../../assets/img/connectors/FileConnector-02.png" title="Adding a Rest API" width="800" alt="Adding a Rest API"/></p>
+
+## Configure the connector in WSO2 Integration Studio
+
+1. Follow these steps to set up the ESB Solution Project and the Connector Exporter Project. 
+[Configure the Connector and the Connector Exporter Project](../configure-connector-exporter-project.md).
+
+2. Right Click on the created ESB Solution Project -> New -> Rest API to create the REST API. 
+<p><img src="../../../assets/img/connectors/adding-an-api.png" title="Adding a Rest API" width="800" alt="Adding a Rest API"/></p>
+
+3. Provide the API name as File Connector and the api context as /fileconnector. You can go to the source view and copy paste the following api configuration. 
```

Comment:
```suggestion
3. Provide the API name as File Connector and the API context as `/fileconnector`. You can go to the source view and copy paste the following API configuration. 
```

Let's put endpoints, and all other technical terms and configurations within single backticks.

Reply:
Done